pkg/scc/scc.go

```diff
@@ -284,6 +438,26 @@ func (builder *Builder) WithUsers(users []string) *Builder {
 	return builder
 }
 
+// WithVolumes adds list of volumes to SecurityContextConstraints.
+func (builder *Builder) WithVolumes(volumes []securityV1.FSType) *Builder {
+	if valid, _ := builder.validate(); !valid {
+		return builder
+	}
+
+	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
+		"volumes: %v", builder.Definition.Name, volumes)
+
+	if builder.Definition.Volumes == nil {
```

Comment:
Please use len validation instead.

Reply:
Sorry, i meant volumes argument and not builder.Definition.Volumes.
Please check function below:
```

// WithVolumes adds list of volumes to SecurityContextConstraints.
func (builder *Builder) WithVolumes(volumes []securityV1.FSType) *Builder {
	if valid, _ := builder.validate(); !valid {
		return builder
	}

	glog.V(100).Infof("Redefining SecurityContextConstraints %s with"+
		"volumes: %v", builder.Definition.Name, volumes)

	if len(volumes) == 0 {
		glog.V(100).Infof("SecurityContextConstraints 'volumes' argument cannot be empty")

		builder.errorMsg = "SecurityContextConstraints 'volumes' cannot be empty list"
		
		return builder
	}
	
	if builder.Definition.Volumes == nil{
		builder.Definition.Volumes = volumes
		
		return builder
	}

	builder.Definition.Volumes = append(builder.Definition.Volumes, volumes...)
	
	return builder
}
```